tests/sign/Makefile

```diff
@@ -0,0 +1,59 @@
+TOP=$(abspath ../..)
```

Comment:
I wonder why this PR has changes already merged into main. The ``./tests/sign`` directory has already been merged.

Reply:
Its because i cherry-picked your commit. It wasnt merged when this PR was created